doc/changes/changes_1.3.1.md

```diff
@@ -0,0 +1,11 @@
+# Virtual Schema for Document Data in Files on Azure Data Lake Storage Gen 2 1.3.1, released 2023-03-24
+
+Code name: Fix performance regression tests
+
+## Summary
+
+This release fixes execution of performance regression tests.
```

Comment:
```suggestion
This release fixes execution of performance regression tests and renames error codes from VSADLG2 to (?).
```

Reply:
We will keep `VSADLG2`. The other code was wrong.